src/app/content/components/Page/PageComponent.tsx

```diff
@@ -29,9 +31,14 @@ export default class PageComponent extends Component<PagePropTypes> {
   private scrollTargetManager = stubScrollTargetManager;
   private processing: Promise<void> = Promise.resolve();
 
-  public getCleanContent = () => {
+  public getTrasnformedContent = () => {
     const {book, page, services} = this.props;
-    return getCleanContent(book, page, services.archiveLoader, contentLinks.reduceReferences(this.props.contentLinks));
+    const cleanContent = parser.parseFromString(
+      getCleanContent(book, page, services.archiveLoader, contentLinks.reduceReferences(this.props.contentLinks)),
+      'text/html'
+    );
+    transformContent(cleanContent.body, this.props.intl);
```

Comment:
the logic inside `transformContent` is depending on the document global and is failing in prerendering, we need to pass in the document that `parseFromString` returns